### PR TITLE
Update Versioning of App Config Packages

### PR DESCRIPTION
**Pull request by mrm9084**
* Updated to 2.0.0 versioning.
* Fixed Spot Bugs issue.
* Fixed samples.

### CODE REVIEW

File: sdk/spring/azure-spring-boot-samples/azure-spring-cloud-sample-appconfiguration-conversion-initial/pom.xml [hunk 1]
```diff
@@ -1,78 +1,62 @@
...
+    <modelVersion>4.0.0</modelVersion>
 
-  <artifactId>azure-spring-cloud-sample-appconfiguration-conversion-initial</artifactId>
-  <groupId>com.azure.spring</groupId>
-  <version>1.0.0</version> <!-- {x-version-update;com.azure.spring:azure-spring-cloud-sample-appconfiguration-convert-initial;current} -->
-  <name>Azure Spring Cloud Sample App Configuration Conversion</name>
-  <description>Sample project to show conversion to Azure App Configuration</description>
+    <artifactId>azure-spring-cloud-sample-appconfiguration-conversion-initial</artifactId>
+    <groupId>com.azure.spring</groupId>
+    <version>2.0.0-beta.1</version> <!-- {x-version-update;com.azure.spring:azure-spring-cloud-sample-appconfiguration-convert-initial;current} -->
```
Comment (saragluna) [inline sdk/spring/azure-spring-boot-samples/azure-spring-cloud-sample-appconfiguration-conversion-initial/pom.xml:15]: Do we need to upgrade the sample version too?

File: sdk/spring/azure-spring-boot-samples/azure-spring-cloud-sample-appconfiguration/pom.xml [hunk 2]
```diff
@@ -11,7 +11,7 @@
 
   <artifactId>azure-spring-cloud-sample-appconfiguration</artifactId>
   <groupId>com.azure.spring</groupId>
-  <version>1.0.0</version> <!-- {x-version-update;com.azure.spring:azure-spring-cloud-sample-appconfiguration;current} -->
+  <version>2.0.0-beta.1</version> <!-- {x-version-update;com.azure.spring:azure-spring-cloud-sample-appconfiguration;current} -->
```
Comment (saragluna) [inline sdk/spring/azure-spring-boot-samples/azure-spring-cloud-sample-appconfiguration/pom.xml:14]: same here

File: sdk/spring/azure-spring-boot-samples/azure-spring-cloud-sample-feature-management-web/pom.xml [hunk 3]
```diff
@@ -11,7 +11,7 @@
 
   <artifactId>azure-spring-cloud-sample-feature-management-web</artifactId>
   <groupId>com.azure.spring</groupId>
-  <version>1.0.0</version> <!-- {x-version-update;com.azure.spring:azure-spring-cloud-sample-feature-management-web;current} -->
+  <version>2.0.0-beta.1</version> <!-- {x-version-update;com.azure.spring:azure-spring-cloud-sample-feature-management-web;current} -->
```
Comment (saragluna) [inline sdk/spring/azure-spring-boot-samples/azure-spring-cloud-sample-feature-management-web/pom.xml:14]: same here

File: sdk/spring/azure-spring-boot-samples/azure-spring-cloud-sample-feature-management-web/src/main/resources/templates/welcome.html [hunk 4]
```diff
@@ -50,7 +50,7 @@
 <div class="container body-content">
     <h1 class="mt-5">Welcome</h1>
     <p>Learn more about <a
-        href="https://github.com/microsoft/spring-cloud-azure/blob/master/azure-spring-cloud-feature-management/README.md">Feature
+        href="https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/spring/azure-spring-boot-samples/feature-management-web-sample/README.md">Feature
```
Comment (saragluna) [inline sdk/spring/azure-spring-boot-samples/azure-spring-cloud-sample-feature-management-web/src/main/resources/templates/welcome.html:53]: Should this point to the same branch?

Comment (mrm9084) [reply sdk/spring/azure-spring-boot-samples/azure-spring-cloud-sample-feature-management-web/src/main/resources/templates/welcome.html:53]: Should be good enough, will be in master soon. The doc is still accurate.

File: sdk/spring/azure-spring-boot-samples/azure-spring-cloud-sample-appconfiguration-conversion-initial/pom.xml (hunk 1 above)
Comment (mrm9084) [reply sdk/spring/azure-spring-boot-samples/azure-spring-cloud-sample-appconfiguration-conversion-initial/pom.xml:15]: I did, not sure what you mean by this.

**Pull request closed without merging**